server/hydrogen-render/1-render-hydrogen-to-string.js

```diff
@@ -60,12 +61,16 @@ async function renderHydrogenToString(options) {
         } else {
           let extraErrorsMessage = '';
           if (childErrors.length > 1) {
-            extraErrorsMessage = ` (somehow we saw ${childErrors.length} errors but we really always expect 1 error)`;
+            extraErrorsMessage = ` (somehow we saw ${
+              childErrors.length
+            } errors but we really always expect 1 error)\n${childErrors
+              .map((childError, index) => ` ${index}. ${childError.message} ${childError.stack}`)
+              .join('\n')}`;
```

Comment:
This will be ugly but I thought that we should give the actually relevant info if this ever happens.